etl/steps/data/garden/happiness/2023-03-20/happiness.meta.yml

```diff
@@ -5,19 +5,35 @@ dataset:
 
     This is called the Cantril ladder: it asks respondents to think of a ladder, with the best possible life for them being a 10 and the worst possible life being a 0. They are then asked to rate their own current lives on that 0 to 10 scale.
 
-    The rankings are calculated by the source based on nationally representative samples for the three years surrounding the study year, so that data for 2021 will draw from survey data from 2020-2022. The number of people and countries surveyed varies year to year, but typically more than 100,000 people in 130 countries participate in the Gallup World Poll each year.
+    The rankings are calculated by the source based on nationally representative samples for the three years prior to the year of the report, so that data for the 2023 report will draw from survey data from 2020-2022. We show the data for final year of the three-year survey period, i.e. we show the 2020-2022 survey data as 2022.
+
+    The only exception is the data for the 2012 report, which uses survey data from 2005-2011, we show this data as the final year of the survey data - 2011.
+
+    The number of people and countries surveyed varies year to year, but typically more than 100,000 people in 130 countries participate in the Gallup World Poll each year.
 
     The rankings are based entirely on the survey scores, using the Gallup weights to make the estimates representative.
 
-    The data for current and previous reports can be found at:
+    Cantril ladder data from for current and previous reports can be found at:
 
     2023: https://happiness-report.s3.amazonaws.com/2023/DataForFigure2.1WHR2023.xls
 
     2022: https://happiness-report.s3.amazonaws.com/2022/Appendix_2_Data_for_Figure_2.1.xls
 
     2021: https://happiness-report.s3.amazonaws.com/2021/DataForFigure2.1WHR2021C2.xls
 
-    Previous years: https://happiness-report.s3.amazonaws.com/2021/DataPanelWHR2021C2.xls
+    2020: https://happiness-report.s3.amazonaws.com/2020/WHR20_DataForFigure2.1.xls
+
+    2019: https://s3.amazonaws.com/happiness-report/2019/Chapter2OnlineData.xls
+
+    2018: https://s3.amazonaws.com/happiness-report/2018/WHR2018Chapter2OnlineData.xls
+
+    2017: https://s3.amazonaws.com/happiness-report/2017/online-data-chapter-2-whr-2017.xlsx
+
+    2016: https://s3.amazonaws.com/happiness-report/2016/Online-data-for-chapter-2-whr-2016.xlsx
+
+    2015: https://s3.amazonaws.com/happiness-report/2015/Chapter2OnlineData_Expanded-with-Trust-and-Governance.xlsx
+
+    2012: https://happiness-report.s3.amazonaws.com/2012/2012.xlsx
```

Comment:
do we need to list the links to the individual datasets in here?

We could instead propagate them from the different dataset's metadata and have them in the `sources` field, right?

Reply:
True - is there a good example of how to do this?